content/blog/self-hosted-search-and-deploy/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Announcing Two New Features for Self-Hosted Pulumi Cloud: Resource Search and Pulumi Deployments"
+allow_long_title: true
+date: 2024-07-02T09:08:31-07:00
+draft: false
+meta_desc: Introducing Resource Search and Pulumi Deployments in Self-Hosted Pulumi Cloud for enhanced management and deployment of cloud infrastructure.
+meta_image: meta.png
+authors:
+    - meagan-cojocar
+tags:
+    - releases
+---
+
+We are excited to introduce two powerful new features for our Self-Hosted Pulumi Cloud offering: Resource Search and Pulumi Deployments. These additions are designed to enhance the way you manage and deploy your cloud infrastructure, providing greater visibility, control, and automation.
+
+## Self-Hosted Pulumi Cloud Overview
+
+Self-Hosting is available in the Pulumi Business Critical edition of Pulumi Cloud, you can try it out with [our 30 day trial](/product/self-hosted). This allows organizations to run the Pulumi Cloud within their own environments, ensuring data control, compliance, and customizability. While Pulumi manages all aspects of its SaaS offering (Pulumi Cloud), Self-Hosted Pulumi Cloud requires customers to handle data backups, service maintenance, and updates.
+
+As of today, all Pulumi Cloud features that have been made generally available are supported in the self-hosted version.
+
+Here’s a deep dive into what the new features offer and how they can benefit your organization:
+
+## Resource Search: Find and Analyze Your Cloud Resources with Ease
+
+Pulumi Resource Search offers multi-cloud search and analytics across every environment in your organization. You can issue queries that find all of your AWS VPCs, or all of your VPCs in AWS and Azure, or all resources with the “production” tag across all cloud environments. You also have access to statistics about cloud usage, including a breakdown by cloud provider, resource type, and department. Resource search enables you to find the needle within your cloud haystack and visualize cloud consumption trends.
+
+![resource search UI](search.png)
+
+### Key Features
+
+- **Multi-Cloud Search:** Query resources across various cloud environments.
+- **Advanced Filtering:** See top stacks, projects, providers, teams, and types by resource count.
+- **AI Assist:** Use natural language queries to generate search syntax.
```

Comment:
Does this work in self-hosted?  Do users have to BYO OpenAI API Key?

Reply:
Guessing no actually- pinged Levi to confirm and will remove.